tools/windows-node-installer/test/e2e/common.go

```diff
@@ -0,0 +1,83 @@
+package e2e
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/masterzen/winrm"
+	"github.com/openshift/windows-machine-config-operator/tools/windows-node-installer/pkg/types"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	// winRMPort is the secure WinRM port
+	winRMPort = "5986"
+	// winFirewallCmd will verify if firewall rule is present on the windows node
+	winFirewallCmd = "cmd /c powershell.exe Get-NetFirewallRule -DisplayName OpenKubeletPort"
```

Comment:
Why are you explicitly running cmd to run powershell? If there is a specific reason to do so, please describe it in a comment. Else, run powershell directly